setup.py

```diff
@@ -59,7 +59,9 @@
     "scikit-learn": ["scikit-learn>=1.3,<1.4"],
     "lightgbm": ["lightgbm>=2,<4"],
     "pytorch": [
+        "requests<3",
```

Comment:
requests is required for proxy support in eland_import_hub_model, so let's declare it explicitly even though transformers requires it already.